.gitignore

```diff
@@ -4,3 +4,4 @@ xcuserdata
 *.xcscmblueprint
 .build/
 Output/
+*.lit_test_times.txt
```

Comment:
This was a drive-by change I added as `lit` kept generating this file and `git` kept marking it as untracked. 

I'm uncertain if this has always been the case, or if I should be leveraging this file. If we _don't_ want to ignore it, I can back this out and consider it a quality-of-life change for my local dev experience.